src/admin-views/settings/tickets-commerce/stripe/signup-link.php

```diff
@@ -4,12 +4,12 @@ class="tec-tickets__admin-settings-tickets-commerce-gateway-signup-settings"
 	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-button">
 		<a
 			data-gateway-onboard-complete="tecTicketsCommerceGatewayStripeSignupCallback"
-			href="<?php echo esc_url( $url ) ?>"
+			href="<?php echo esc_attr( $url ) ?>"
 			data-gateway-button="true"
 			id="connect_to_stripe"
 			class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-button-link"
 		>
-			<?php echo wp_kses( __( 'Connect with <i>Stripe</i>', 'event-tickets' ), 'post' ); ?>
+			<?php echo wp_kses_post( __( 'Get Connected with <i>Stripe</i>', 'event-tickets' ) ); ?>
```

Comment:
Can we do the following?

```suggestion
			<?php echo wp_kses( __( 'Get Connected with <i>Stripe</i>', 'event-tickets' ), [ 'i'=> [], 'em' => [], 'strong' => [] ] ); ?>
```